lib/git_reflow/commands/refresh.rb

```diff
@@ -0,0 +1,25 @@
+desc 'refresh will merge your current branch based on the base branch'
```

Comment:
No need to specify the command name in the desc as it is auto generated. Also the description is misleading. Let's go with something like "Updates and synchronizes your base branch and feature branch."